turtlebot2_drivers/launch/turtlebot_carto_3d.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+from launch.exit_handler import restart_exit_handler
+
+
+def launch(launch_descriptor, argv):
+    ld = launch_descriptor
+    ld.add_process(
+        cmd=['kobuki_node'],
+        name='kobuki_node',
+        exit_handler=restart_exit_handler,
+    )
+    ld.add_process(
+        cmd=['astra_camera_node', '-dw', '320', '-dh', '240'],
+        name='astra_camera_node',
+        exit_handler=restart_exit_handler,
+    )
+    ld.add_process(
+        cmd=['depth_to_pointcloud_node'],
+        name='depth_to_pointcloud_node',
```

Comment:
We will do that after the beta I guess but it would be great if the package/executable names were consistent: depthimage_to_laserscan <=> depth**image**_to_pointcloud